httpreplay/cobweb.py

```diff
@@ -25,7 +28,8 @@ def _read_chunked(rfile):
             try:
                 length = int(line, 16)
             except ValueError:
-                raise dpkt.UnpackError("Invalid chunked encoding length: {}".format(line))
+                raise dpkt.UnpackError(
+                    "Invalid chunked encoding length: {}".format(line))
```

Comment:
While you're at it, please replace the `"{}"` syntax by a simple `"%s"` and put the trailing closing parenthesis on the next line.